crates/bevy_text/src/text2d.rs

```diff
@@ -125,42 +127,36 @@ pub fn extract_text2d_sprite(
 
 #[derive(Debug, Default)]
 pub struct QueuedText2d {
-    entities: Vec<Entity>,
+    entities: HashSet<Entity>,
 }
```

Comment:
This struct is only used locally in the text2d_system below, but is actually `pub` and thus visible to users of Bevy.
I suggest removing it entirely, and just using a `Local<HashSet<Entity>>` directly, but at the very least it should not be `pub`.